case_utils/inherent_uuid.py

```diff
@@ -96,6 +96,15 @@
 )
 
 
+def dictionary_entry_inherence_uuid(
+    uco_object_uuid_namespace: uuid.UUID, key_name: str, *args: Any, **kwargs: Any
```

Comment:
Why accept `*args` and `**kwargs` if they're silently ignored?

Reply:
Fair point.  These were my suggestion, so I'll field the question.

This addresses a syntax in Python's function definitions that frequently looks ambiguous to me, where a kind-of-positional spelling can be used at call time.

I've found it helpful to have at least `*args` always, as a reminder that there's a point in the argument list where the arguments are no longer positional.  If it comes at the end, great!  All arguments are positional.  `**kwargs` tends to come along as a matter of habit - if there's `*args`, I tend to want `**kwargs` as well.

I also include `*args` and `**kwargs` in class methods when I know subclasses are going to override the implementations, or use `super()`.

So, by two habits, I use `*args` and `**kwargs` in function definitions, and I suggested those to @sldouglas-nist .